metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran_query.py

```diff
@@ -1,76 +1,74 @@
 class FivetranLogQuery:
-    @staticmethod
-    def use_schema(db_name: str, schema_name: str) -> str:
-        return f'use schema "{db_name}"."{schema_name}"'
+    def __init__(self) -> None:
+        # Select query db clause
+        self.db_clause: str = ""
 
-    @staticmethod
-    def get_connectors_query() -> str:
-        return """
-        SELECT connector_id as "CONNECTOR_ID",
-        connecting_user_id as "CONNECTING_USER_ID",
-        connector_type_id as "CONNECTOR_TYPE_ID",
-        connector_name as "CONNECTOR_NAME",
-        paused as "PAUSED",
-        sync_frequency as "SYNC_FREQUENCY",
-        destination_id as "DESTINATION_ID"
-        FROM CONNECTOR
+    def set_db(self, db_name: str) -> None:
+        self.db_clause = f"{db_name}."
+
+    def use_database(self, db_name: str) -> str:
+        return f"use database {db_name}"
+
+    def get_connectors_query(self) -> str:
+        return f"""
+        SELECT connector_id,
+        connecting_user_id,
+        connector_type_id,
+        connector_name,
+        paused,
+        sync_frequency,
+        destination_id
+        FROM {self.db_clause}connector
         WHERE _fivetran_deleted = FALSE"""
 
-    @staticmethod
-    def get_user_query(user_id: str) -> str:
+    def get_user_query(self, user_id: str) -> str:
         return f"""
-        SELECT id as "USER_ID",
-        given_name as "GIVEN_NAME",
-        family_name as "FAMILY_NAME"
-        FROM USER
+        SELECT id as user_id,
+        given_name,
+        family_name
+        FROM {self.db_clause}user
         WHERE id = '{user_id}'"""
 
-    @staticmethod
-    def get_sync_start_logs_query(
-        connector_id: str,
-    ) -> str:
+    def get_sync_start_logs_query(self, connector_id: str) -> str:
         return f"""
-        SELECT time_stamp as "TIME_STAMP",
-        sync_id as "SYNC_ID"
-        FROM LOG
+        SELECT time_stamp,
+        sync_id
+        FROM {self.db_clause}log
         WHERE message_event = 'sync_start'
         and connector_id = '{connector_id}' order by time_stamp"""
 
-    @staticmethod
-    def get_sync_end_logs_query(connector_id: str) -> str:
+    def get_sync_end_logs_query(self, connector_id: str) -> str:
         return f"""
-        SELECT time_stamp as "TIME_STAMP",
-        sync_id as "SYNC_ID",
-        message_data as "MESSAGE_DATA"
```

Comment:
does this work across bigquery and snowflake? are the output column names the same

Reply:
yes it works. I cross checked it.